Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -94,19 +83,22 @@ public class UserManager : IUserManager
             IJsonSerializer jsonSerializer,
             IFileSystem fileSystem)
         {
-            _logger = loggerFactory.CreateLogger(nameof(UserManager));
-            UserRepository = userRepository;
+            _logger = logger;
+            _userRepository = userRepository;
             _xmlSerializer = xmlSerializer;
             _networkManager = networkManager;
             _imageProcessorFactory = imageProcessorFactory;
             _dtoServiceFactory = dtoServiceFactory;
             _appHost = appHost;
             _jsonSerializer = jsonSerializer;
             _fileSystem = fileSystem;
-            ConfigurationManager = configurationManager;
-            _users = Array.Empty<User>();
+            _users = null;
```

Comment:
Why null?

Reply:
Because the “Initialize” function creates an already populated dictionary.

Reply:
I would prefer to stick to the old approach of creating an empty structure. We have less chance of getting a nice null ptr exception.

Reply:
It's practically impossible since the `Initialization` function is called almost directly after creation.